lib/appsignal/config.ex

```diff
@@ -214,6 +214,9 @@ defmodule Appsignal.Config do
     unless empty?(config[:files_world_accessible]) do
       System.put_env("_APPSIGNAL_FILES_WORLD_ACCESSIBLE", to_string(config[:files_world_accessible]))
     end
+    unless empty?(config[:revision]) do
+      System.put_env("APP_REVISION", to_string(config[:revision]))
+    end
```

Comment:
This env var should always be set, otherwise the value would persist across releases with the rolling restarts.

But the same problem probably exists for other config options as well.